### PR TITLE
ar.json

### PR DESCRIPTION
**Pull request by itsBarham**
Barham Kamal #5916

### CODE REVIEW

**Comment by njb-said** [comment]
Unfortunately I'll have to hold off on this one until I've made the site flippable to read from right to left 😞 
Hopefully I can do that ASAP!

**Comment by BarhamKamal** [comment]
In fact I have some experience in web development I can help you change direction from left to right :)

File: ar.json [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+      "api" : "api",
+      "account" : {
+        "dropdown" : "الحساب",
+        "account" : "الحساب",
+        "logout" : "الخروج"
+      },
+      "languagePicker" : {
+        "trigger" : "اللغة",
+        "title" : "اختر اللغة",
+        "currentLanguage" : "أنت الآن تستخدم العربية."
```
Comment (Krymonota) [inline ar.json]: Please don't translate placeholders. `[language]` is missing.

Comment (BarhamKamal) [reply ar.json]:
```suggestion
        "currentLanguage" : "أنت الآن تستخدم [language]."
```
like this?

**Comment by Krymonota** [comment]
See #125 for a more recent and complete version.

**Pull request closed without merging**